sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -51,11 +52,12 @@ pkg_setup() {
 
 src_prepare() {
 	cp -f "${FILESDIR}/Makefile" lib/ || die
+	use elibc_musl && epatch "${FILESDIR}/${P}-musl-support.patch"
```

Comment:
here you could use ``append-cppflags`` it seems;


rest of the patch lgtm, thanks